packages/profile-flyout/src/presenters/ProfileContent.scss

```diff
@@ -0,0 +1,44 @@
+@import "~@hig/styles/mixins/base.scss";
+
+.hig__profile-flyout__content {
+  @include reset;
+
+  padding: 10px;
+  font-family: ArtifaktElement;
+  line-height: 1.6;
+  padding: 10px;
+
+  &__name {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    font-size: 16px;
+    line-height: 24px;
+    color: #222934;
+  }
+
+  &__email {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    color: #565c5f;
+    line-height: 16px;
+    font-size: 12px;
+  }
+
+  &__links {
```

Comment:
Are `__links` and `__link` used?

Reply:
They are used in the styling of the children that are passed to `ProfileContent`. I'm guessing it would make more sense to remove that here and leave the styling up to the devs using the component.

Reply:
I agree. I would leave any other styling up to the client dev. 👍

Reply:
That makes sense. Would you like the storybook example to still have those two buttons, but with the styles refactored? I was thinking having some filler content like `Any custom content could go here` might make things more clear.

Reply:
That implementation looks like this in Gemini:

<img width="428" alt="screen shot 2018-09-11 at 10 25 29 am" src="https://user-images.githubusercontent.com/9288982/45366774-a1018080-b5ad-11e8-9619-3309481bf833.png">

I'm not sure if that width and the text wrapping is what is desired visually. What do you think?